File: Loadshop.DomainServices/Loadshop/Services/Data/LoadStatusNotificationsData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Loadshop.DomainServices.Loadshop.Services.Data
{
    public class LoadStatusNotificationsData
    {
        public bool TextMessageEnabled { get; set; }
        public string TextMessageNumber { get; set; }
        public bool EmailEnabled { get; set; }
        public string Email { get; set; }
        public bool DepartedEnabled { get; set; }
        public bool ArrivedEnabled { get; set; }
        public bool DeliveredEnabled { get; set; }
    }
}

File: Loadshop.Web.UI/src/app/shared/models/load-status-notifications-data.ts
export interface LoadStatusNotificationsData {
  textMessageEnabled: boolean;
  textMessageNumber: string;
  emailEnabled: boolean;
  email: string;
  departedEnabled: boolean;
  arrivedEnabled: boolean;
  deliveredEnabled: boolean;
}

export const defaultLoadNotification: LoadStatusNotificationsData = {
  textMessageEnabled: false,
  textMessageNumber: '',
  emailEnabled: false,
  email: '',
  departedEnabled: false,
  arrivedEnabled: false,
  deliveredEnabled: false,
};
